modules/core/src/com/haulmont/cuba/security/app/UserSettingServiceBean.java

```diff
@@ -166,7 +166,7 @@ public void copySettings(User fromUser, User toUser) {
     }
 
     @Override
-    public void deleteAllScreensSettings(ClientType clientType) {
+    public void deleteAllScreenSettings(ClientType clientType) {
         List<String> exclude = Arrays.asList("appWindowMode", "appWindowTheme", "userDefaultScreen");
```

Comment:
We should try to find another solution here or just drop all the settings instead.